aten/src/ATen/native/native_functions.yaml

```diff
@@ -2173,6 +2173,36 @@
 - func: isclose(Tensor self, Tensor other, float rtol=1e-05, float atol=1e-08, bool equal_nan=False) -> Tensor
   variants: function, method
 
+- func: isin.Tensor_Tensor_out(Tensor elements, Tensor test_elements, bool assume_unique=False, bool invert=False, *, Tensor(a!) out) -> Tensor(a!)
```

Comment:
In the Python Array API, which is our new compatibility target, all argument are either positional-only or kwarg-only. PyTorch doesn't support positional-only arguments (yet), but we do suppot kwarg-only. What would you think about making `assume_unique` and `invert` kwarg-only?

Reply:
Sure, sounds good! I assume that just entails moving the `*` slightly over?